nservicebus/upgrades/7to8/dependency-injection.md

```diff
@@ -154,6 +154,21 @@ var autofacContainer = containerBuilder.Build();
 var endpointInstance = await startableEndpoint.Start(new AutofacServiceProvider(autofacContainer));
 ```
 
+With newer versions of Autofac the [ability to update an existing container](https://github.com/autofac/Autofac/issues/811) has been removed. The registration phase of the dependencies is now seperate from the resolve phase. Once a component has been resolved the container is immutable. For cases when an existing lifetime scope was passed to NServiceBus
+
+```csharp
+endpointConfiguration.UseContainer<AutofacBuilder>(
+    customizations: customizations =>
+    {
+        customizations.ExistingLifetimeScope(container);
+    });
+```
+
+it is recommended to either
+
+- Stop building the container early and pass around the container builder to the various bootstrapping parts of the application _or_
+- Consider creating a [dedicated child lifetime scope](https://autofac.readthedocs.io/en/latest/integration/netcore.html#using-a-child-scope-as-a-root)
```

Comment:
```suggestion
- Using a [dedicated child lifetime scope](https://autofac.readthedocs.io/en/latest/integration/netcore.html#using-a-child-scope-as-a-root)
```

Reply:
Nitpick: Perhaps end with what this achieves? ( Is it to "make sure that XYZ is the root container"?)